pkg/uploader/kopia/snapshot.go

```diff
@@ -65,6 +66,10 @@ func newOptionalInt(b policy.OptionalInt) *policy.OptionalInt {
 	return &b
 }
 
+func newOptionalBool(b policy.OptionalBool) *policy.OptionalBool {
```

Comment:
IMO it would be slightly better if the input param is changed to `bool`, like 
```
func newOptionalBool(b bool) *policy.OptionalBool {
	ob := policy.OptionalBool(b)
	return &ob
}
```
otherwise, you are not creating anything `new` in this func

Reply:
yeah, Also I modified `newOptionalInt` function